lua/pac3/core/client/parts/interpolated_multibone.lua

```diff
@@ -0,0 +1,230 @@
+local BUILDER, PART = pac.PartTemplate("base_drawable")
+
+PART.ClassName = "interpolated_multibone"
```

Comment:
Can we name this something like "part interpolation"? Initially I thought it had something to do with bones

Reply:
I can't be the only one who started using these. Changing the classname is a big decision to make.

Maybe use a FriendlyName field to rename it? But the idea with multibone is that it can ultimately move between bones. Bones being the origin of all base_movables, it was a serious limitation that we couldn't smoothly shift a part between them. The choice of name doesn't really matter though, but undoubtedly the goal is to get the point across.

here's more ideas. tell me if one strikes your fancy.
path
interpolator
position_morpher